target/linux/ath79/dts/ar9344_netgear_r6100.dts

```diff
@@ -0,0 +1,239 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	compatible = "netgear,r6100", "qca,ar9344";
+	model = "Netgear R6100";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_power_amber;
+		led-failsafe = &led_power_amber;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_amber;
+		label-mac-device = &eth0;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "wps";
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+		};
+
+		rfkill {
+			label = "rfkill";
+			linux,code = <KEY_RFKILL>;
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&enable_gpio_11>;
+
+		led_power_green: power_green {
+			label = "green:power";
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_amber: power_amber {
+			label = "amber:power";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			default-state = "keep";
+		};
+
+		led_wlan_blue {
+			label = "blue:wlan";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		led_usb_blue {
+			label = "blue:usb";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&hub_port>;
+			linux,default-trigger = "usbport";
+		};
+
+		wan_green {
+			label = "green:wan";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+
+		wan_amber {
+			label = "amber:wan";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&gpio {
+	gpio_usb_power: usb_power {
+		gpio-hog;
+		gpios = <16 0>;
+		output-high;
+	};
+};
+
+&pinmux {
+	enable_gpio_11: pinmux_enable_gpio_11 {
+		pinctrl-single,bits = <0x8 0x0 0xff000000>;
+	};
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "u-boot";
+			reg = <0x0 0x20000>;
+			read-only;
+		};
+
+		caldata: partition@20000 {
+			label = "caldata";
+			reg = <0x20000 0x40000>;
+			read-only;
+		};
+
+		partition@60000 {
+			label = "caldata-backup";
+			reg = <0x60000 0x40000>;
+			read-only;
+		};
+
+		partition@a0000 {
+			label = "config";
+			reg = <0xa0000 0x80000>;
+		};
+
+		partition@120000 {
+			label = "pot";
+			reg = <0x120000 0x80000>;
+		};
+
+		kernel@1a0000 {
+			label = "kernel";
+			reg = <0x1a0000 0x400000>;
+		};
+
+		ubi@5a0000 {
```

Comment:
partition@ for these two as well?

Reply:
ar9344_netgear_wndr.dtsi has ubi@ and kernel@ in the DTS too, not partition@ - I thought about following that, but can change if required.

Reply:
The general recommendation is to use generic names there, i.e. just partition@ for all partitions in this case. Of course, you need to have the label property set then. If something different than partition@ is used, it's either old or slipped through review AFAIK.